src/components/CodeEditor/CodeEditor.js

```diff
@@ -35,6 +35,8 @@ class CodeEditor extends Component {
   componentDidUpdate(prevProps, prevState) {
     if (prevState.compiled !== this.state.compiled) {
       this._render();
+    } else if (this.props.code !== prevProps.code) {
```

Comment:
Added this so that if _new_ props come in (i.e. home/examples/simple-example.js is updated) the snippet updates cleanly.